application/defs/include/osib_example.yml

```
@@ -0,0 +1,361 @@
+####################################################################################
+# osib_short.yaml: Open Security Information Base (OBIB), (c) OWASP Top 10 team    #
+# This work is licendes under a Creative Commons Attribution 3.0 Unported Licence  #
+# ---------------------------------------------------------------------------------#
+# Example File                                                                     #
+####################################################################################
+
+doctype:        OSIB
+schema:         90         # V 0.90
+date:           20211104
+children:
+  2:
```

hey @sslHello if we have multiple children this needs to be a list, right?
so the line should be the following

```suggestion
-  2:
```

---

As for now a children are nested dictionaries right now, same for 'sources_i18n'. 
In my opinion, it is easier and more efficient to adress items directly using a directory than a list. Lists are a kind of step in the dark, as you don't know the items and you have to lookup them item by item to find the right element...
How about using dictionaries for nodes and elements that can ge precisely addess by a path e.g. osib.owasp.top10.2021-1-0.1 = Top10, 2021, V1.0, A01:2021?

---

> In my opinion, it is easier and more efficient to address items directly using a directory than a list. 
Fair, will keep it a Dict[str, Osib_node] then